package.json

```diff
@@ -1,9 +1,15 @@
 {
   "name": "kizuna-platform",
-  "version": "1.0.0",
+  "version": "0.0.1",
   "description": "Friendship Game.  For Vote, Rank,  QA ...",
   "main": "index.js",
   "scripts": {
+    "build": "npm run clean && webpack --config ./webpack.prod.config.js",
+    "clean": "rm -rf build",
+    "eslint": "eslint src test",
+    "lint": "npm run eslint && npm run stylelint",
+    "start": "webpack-dev-server --devtool eval --progress --colors --hot --history-api-fallback --content-base dist",
+    "stylelint": "stylelint \"src/**/*.scss\" --syntax scss",
```

Comment:
I suggest make the following changes:

Change `` eslint `` to `` ./node_modules/.bin/eslint ``
Change `` webpack-dev-server `` to `` ./node_modules/.bin/webpack-dev-server ``
Change `` stylelint `` to `` ./node_modules/.bin/stylelint ``